aldryn_people/models.py

```diff
@@ -111,7 +117,11 @@ class Meta:
         verbose_name_plural = _('People')
 
     def __str__(self):
-        return self.name
+        pkstr = str(self.pk)
+
+        if six.PY2:
+            pkstr = six.u(pkstr)
+        return self.name.strip() or pkstr
```

Comment:
I can change it like

```python
name = self.name.strip()
return name or pkstr
```

but don't think is necessary to do if:return else:return